app/imports/components/ActionItems/CloseOrReopenActionItem/CloseOrReopenActionItem.jsx

```diff
@@ -39,10 +41,12 @@ const CloseOrReopenActionItem = ({
             >
                 <ErrorsField />
                 { isOpen ?
-                    <TextField
+                    <CustomTextField
                         name="closeMessage"
                         floatingLabelText="Close message - optional"
                         fullWidth
+                        value={closeMessage}
```

Comment:
`value={closeMessage}` overwrite your message when you want to change it, because you can't change prop in this way.